x-pack/plugins/apm/scripts/jest.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+// eslint-disable-next-line import/no-extraneous-dependencies
+require('@babel/register')({
+  extensions: ['.js'],
+  plugins: [],
+  presets: [
+    '@babel/typescript',
+    ['@babel/preset-env', { targets: { node: 'current' } }],
+  ],
+});
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { run } = require('jest');
+
+process.env.NODE_ENV = process.env.NODE_ENV || 'test';
+
+const config = require('../jest.config.js');
+
+const argv = [...process.argv.slice(2), '--config', JSON.stringify(config)];
```

Comment:
Looks like this could/should be a path
![image](https://user-images.githubusercontent.com/209966/93753797-ff33b600-fc00-11ea-8374-cd97e44d2d64.png)

```suggestion
const argv = ['--config', '../jest.config.js'];
```

Reply:
It's the same approach as the top-level jest test runner uses.